ui/keys/keys.go

```diff
@@ -158,3 +159,52 @@ var Keys = KeyMap{
 		key.WithHelp("q", "quit"),
 	),
 }
+
+// Rebind will update our saved keybindings from configuration values.
+func Rebind(universal, issueKeys, prKeys []config.Keybinding) error {
+	err := rebindUniversal(universal)
+	if err != nil {
+		return err
+	}
+
+	err = rebindPRKeys(prKeys)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func rebindUniversal(universal []config.Keybinding) error {
+	for _, kb := range universal {
+		log.Debug("Rebinding key", "builtin", kb.Builtin, "key", kb.Key)
+		switch kb.Builtin {
+		case "up":
```

Comment:
I *think* you can use the keys.Keys global var here instead of the switch case. 
If you don't like updating a global var, you can expose a method in the `keys` package. I think this will be automatically picked up in the ui.go file

Reply:
The switch is used to map the yaml string to the keybinding in the global var. I am not sure how to go bind the two otherwise. Other then maybe reflection.